src/components/executive-education-2u/data/utils.js

```diff
@@ -4,3 +4,5 @@ export const toISOStringWithoutMilliseconds = (isoString) => {
   }
   return `${isoString.split('.')[0]}Z`;
 };
+
+export const isDuplicateOrder = (formSubmissionError) => formSubmissionError?.message?.includes('duplicate order');
```

Comment:
nit: might recommend naming this as `isDuplicateExternalCourseOrder` to be a tad bit more explicit